assets/src/scss/layout/_navbar.scss

```diff
@@ -204,6 +204,93 @@ $navbar-default-height: 60px;
   }
 }
 
+.nav-submenu {
+  position: fixed;
+  display: none;
+  min-width: 193px;
+  padding-top: $sp-1;
+
+  .nav-submenu-wrapper {
+    border-radius: $sp-1;
+    width: 100%;
+    box-shadow: 0 1px $sp-x rgba(0, 0, 0, 0.12);
+  }
+
+  ul {
+    overflow: hidden;
+    flex-direction: column;
+    padding: $sp-2 0;
+    width: 100%;
+    border-radius: $sp-1;
+    background: $white;
+
+    &.scrollable {
+      height: 320px;
+      overflow-y: auto;
+      display: block;
+
+      // Only applied to Firefox
+      scrollbar-width: thin;
+      scrollbar-color: transparent;
+      scrollbar-gutter: stable both-edges;
```

Comment:
This one I actually see applied in Chrome as well, and it looks a bit strange I think:
<img width="305" alt="Screenshot 2022-02-24 at 09 44 37" src="https://user-images.githubusercontent.com/6949075/155489718-1f3cf2ff-efd6-4347-9086-eed0c81e4dcb.png">